SimPEG/Regularization.py

```diff
@@ -1,12 +1,33 @@
 from __future__ import print_function
-from . import Utils
-from . import Maps
-from . import Mesh
+
 import numpy as np
 import scipy.sparse as sp
+import warnings
+import properties
 
-
-class RegularizationMesh(object):
+from . import Utils
+from . import Maps
+from . import Mesh
+from . import ObjectiveFunction
+from . import Props
+
+__all__ = [
+    'Smallness',
+    'Smooth_x', 'Smooth_y', 'Smooth_z',
+    'Smooth_xx', 'Smooth_yy', 'Smooth_zz',
```

Comment:
I think these should be turned from classes into inputs. `Smooth(orientation='x')` `Smooth2` etc.